packages/build-ui/src/pages/collection/page.rs

```diff
@@ -0,0 +1,320 @@
+use bdk::prelude::*;
+use dioxus_translate::Language;
+use by_components::icons::{ settings, folder, edit };
+
+use crate::pages::collection::{Artwork, Collection, CollectionNameModal, FilterSidebar, NewCollectionModal, SuccessModal, TransferConfirmationModal};
+
+
+#[derive(Clone, Copy, PartialEq)]
+#[warn(dead_code)]
+enum ModalState {
+    None,
+    NewCollection,
+    Transfer,
+    CollectionName,
+    Success
+}
+
+#[allow(unused_variables)]
+#[component]
+pub fn CollectionsPage(lang: Language, agit_id:i64) -> Element {
```

Comment:
```suggestion
pub fn CollectionsPage(lang: Language, agit_id: ReadOnlySignal<i64>) -> Element {
```
Dioxus will automatically convert to ReadOnlySignal.
If you don't use `ReadOnlySignal`, it causes a bug when navigating from /collections/1 to  /collections/2.

And you may not set formatter.